### ISSUE TITLE
Example to use with pytorch

### ISSUE THREAD

**Issue by ruathudo** [JenEskimos/soft_edit_distance#1]
Hi, this is interesting. I want to use the soft edit distance as the metric for loss function in pytorch.
Do you have any example to use it with pytorch? 
Thanks!

**Comment by williamking5** [comment]
I have the same issue

**Comment by williamking5** [comment]
It seems that the implementation of chainer.cuda is very different from torch

**Comment by prometheusred** [comment]
I was actually thinking about trying to implement this in pytorch.  If anyone has any tips, much appreciated and would be happy to share.
